nengo_viz/static/viz_slider.js

```diff
@@ -152,8 +164,8 @@ VIZ.Slider.prototype.constructor = VIZ.Slider;
 
 VIZ.Slider.prototype.set_value = function(slider_index, value, immediate) {
     console.assert(typeof slider_index == 'number');
-    console.assert(typeof value == 'number');
-    console.assert(typeof immediate == 'boolean');
+    console.assert(typeof value == 'number', value);
+    //console.assert(typeof immediate == 'boolean');
```

Comment:
Actually, that whole argument is going to disappear and get reorganized when #160 gets brought in.  So I'm tempted to just put a TODO in here.